src/Components/LandingPage.jsx

```diff
@@ -1,29 +1,40 @@
 import React from 'react';
 import styles from './LandingPage.module.css';
```

Comment:
Check your import order:
```suggestion
import React from 'react';

import CurrentDate from './Date';
import StartAstroPage from './ModuleAstro';
import StartMeteoPage from './ModuleMeteo';
import StartShopsPage from './ModuleShops';
import StartSNCFPage from './ModuleSNCF';
import AdressSearch from './AdressSearch';
import Nav from '../components/Nav';

import styles from './LandingPage.module.css';
```